Add-ons/UmbracoForms/Developer/Rendering-Scripts/index.md

```diff
@@ -43,6 +76,11 @@ To enable `ExcludeScripts`:
 `ExcludeScripts = "1"` prevents the associated scripts from being rendered. Any other value, an empty value, or if the parameter is excluded, will render the scripts on the Form.
 :::
 
+The `Scripts.cshtml` file (from the current or default theme) will not be initiated and won't be included as part of the current form. You may however use parts (or all) of the code from the snippet of `Scripts.cshtml` within your own template or partial views.
+
+
+In many cases, you might prefer rendering your scripts at the bottom of the page, e.g. before the closing `</body>` tag, as this generally improves site performance.
```

Comment:
```suggestion
In many cases, you might prefer rendering your scripts at the bottom of the page, for example before the closing `</body>` tag. This will generally improve site performance.
```